src/app/docs/inline-delete/inline-delete-docs.component.html

```diff
@@ -0,0 +1,99 @@
+<sky-docs-demo-page
+  moduleName="SkyInlineDeleteModule"
+  moduleSourceCodePath="src/app/public/modules/inline-delete/"
+  pageTitle="Inline delete"
+>
+  <sky-docs-demo-page-summary>
+    Sample description.
```

Comment:
Yup that sounds good. I do wonder if we need to refer to it as "inline delete confirmation" instead of "inline delete"?

Reply:
I agree. I think that makes sense throughout the docs. I just didn't make that suggestion because I was waiting to see the response on my question about adding "confirmation" to the component name.